core/src/test/java/com/graphhopper/routing/util/DataFlagEncoderTest.java

```diff
@@ -346,7 +345,11 @@ public void testSpatialId() {
         e1.setFlags(encoder.handleWayTags(way, 1, 0));
         e2.setFlags(encoder.handleWayTags(way2, 1, 0));
 
-        assertEquals(encoder.getSpatialId(e1.getFlags()), encoder.getSpatialId(e1.getFlags()));
-        assertEquals(index.getSpatialId(EMPTY), encoder.getSpatialId(e2.getFlags()));
+        assertEquals(index.getSpatialId(new GermanySpatialRule()), encoder.getSpatialId(e1.getFlags()));
+        assertEquals(index.getSpatialId(SpatialRule.EMPTY), encoder.getSpatialId(e2.getFlags()));
+
+        assertEquals(AccessValue.EVENTUALLY_ACCESSIBLE, encoder.getAccessValue(e1.getFlags()));
+        assertEquals(AccessValue.ACCESSIBLE, encoder.getAccessValue(e2.getFlags()));
```

Comment:
Would you mind to add a further test where the max_speed value is overwritten by the one stored in the index?